validator/sawtooth_validator/server/core.py

```diff
@@ -178,7 +178,6 @@ def __init__(self, bind_network, bind_component, endpoint,
         block_store = BlockStore(block_db)
 
         batch_tracker = BatchTracker(block_store)
-        block_store.add_update_observer(batch_tracker)
```

Comment:
Is it worth keeping the `add_update_observer` method and the `StoreUpdateObserver` class around if nothing uses it? They should probably be removed.